src/sections/Community/Handbook/repository.js

```diff
@@ -2,9 +2,16 @@ import React from "react";
 import { Container, Row, Col } from "../../../reusecore/Layout";
 import { HandbookWrapper } from "./Handbook.style";
 import TOC from "../../../components/handbook-navigation/index";
+import { repo_data } from "./repo-data";
 
-
-const Repository= () => {
+const Repository = () => {
+  const data = React.useMemo(() => repo_data);
+  let frontend =  data.filter((data) => data.category === "Frontend");
```

Comment:
This is creating unnecessary computation. You can clean up the code. Create an array like - 

```
repo_data = [  
  {
    category: "Frontend",
    data: [
    {
    project: "Layer5",
    site: "https://layer5.io/", 
    language: "Gatsby and Strapi", 
    repository:  "https://github.com/layer5io/layer5" , 
    },
    {
    site: "https://meshery.io", 
    language: "Jekyll", 
    repository:  "https://github.com/meshery/meshery" , 
    },
   ]
  },
  {
    category: "Installation Dedicated Repos",
    data: [......]
  },
]
```